README.md

```diff
@@ -45,6 +45,29 @@ Currently just two SQL dialects are supported:
 - **sql** - [Standard SQL][]
 - **n1ql** - [Couchbase N1QL][]
 
+### Placeholders replacement
+
+```js
+// Named placeholders
+sqlFormatter.format("SELECT @foo FROM table1", {
+    params: {foo: "bar"}
+}));
+
+// Indexed placeholders
+sqlFormatter.format("SELECT ? FROM table1", {
+    params: ["bar"]
+}));
+```
```

Comment:
We should use a more real-life example. One normally uses placeholders in conditions of WHERE clause like:

```
SELECT * FROM tbl WHERE foo = ?
```